modules/apps/transfer/keeper/grpc_query_test.go

```diff
@@ -91,12 +92,12 @@ func (suite *KeeperTestSuite) TestQueryDenom() {
 
 			res, err := suite.chainA.GetSimApp().TransferKeeper.Denom(ctx, req)
 
-			if tc.expPass {
+			if tc.expErr == nil {
 				suite.Require().NoError(err)
 				suite.Require().NotNil(res)
 				suite.Require().Equal(&expDenom, res.Denom)
 			} else {
-				suite.Require().Error(err)
+				ibctesting.RequireErrorIsOrContains(suite.T(), err, tc.expErr, err.Error())
```

Comment:
Maybe in a separate PR we should update other tests where this function could be used... More refactoring of tests, yay!!

Reply:
Definitely! But I think after these I'll take a break from refactoring tests :D